templates/home.macros.j2

```diff
@@ -38,3 +38,23 @@
     </span>
   </span>
 {% endmacro %}
+
+
+{% macro expertise(
```

Comment:
@mirisuzanne how do I get this to render inside the rst block (content-block and content-block-inner)?

Reply:
you should be able to do the rst `start`, and `end` on either side:

```njk
.. callmacro:: content.macros.j2#rst
  :tag: 'start'

.. callmacro:: home.macros.j2#rstexpertise
  <!-- etc -->

.. callmacro:: content.macros.j2#rst
  :tag: 'end'
```

Reply:
hmmm, I thought I tried that but I will do it again. I probably had something wrong, like indentation or something silly

Reply:
@stacyk the `rst` macro can be used in two ways:

```
.. callmacro:: content.macros.j2#rst
    <!-- without a "tag" it requires indenting nested content -->

.. callmacro:: content.macros.j2#rst
  :tag: 'start'

<!-- between "start" and "end" tags, nested content should not indent -->

.. callmacro:: content.macros.j2#rst
  :tag: 'end'
```